packages/content-api/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@tryghost/content-api",
+  "version": "0.0.0",
+  "repository": "https://github.com/TryGhost/Ghost-SDKs/tree/master/packages/content-api",
+  "author": "Ghost Foundation",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "dev": "echo \"Implement me!\"",
+    "test": "NODE_ENV=testing mocha './test/**/*.test.js'",
+    "lint": "eslint . --ext .js --cache",
+    "posttest": "yarn lint"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "mocha": "5.2.0",
+    "should": "13.2.3",
+    "sinon": "7.0.0"
+  },
+  "dependencies": {
+    "bluebird": "^3.5.2",
+    "ghost-ignition": "^2.9.6",
```

Comment:
Are bluebird and ghost-ignition used? Something to watch for when we implement the build process to ensure they aren't pulled in unnecessarily.

Reply:
Nice thanks :+1:

Reply:
Removed bluebird, we _might_ end up using errors from ignition, not sure yet